packages/mesos-modules/build

```diff
@@ -1,15 +1,53 @@
 #!/bin/bash
+set -o nounset -o pipefail
+
+# Update the image and download build tools.
+apt-get update -qq
+apt-get install -y autoconf libtool make pkg-config
+
+# Use an older version of GCC (older than the one in `build-essential`)
+# so that we're ABI-compatible with CentOS 7.
+apt-get install -y gcc-4.8 g++-4.8
```

Comment:
the GCC version is not what matters here. We need to use the same build environment top to bottom. There's a lot that can cause binary incompatiblities outside of gcc version.